rclcpp/src/rclcpp/node_interfaces/node_base.cpp

```diff
@@ -20,6 +20,7 @@
 #include "rclcpp/node_interfaces/node_base.hpp"
 
 #include "rcl/arguments.h"
+#include "rcl/logging.h"
```

Comment:
Sorry, I missed it.

It's better to add `#include <rcl/logging_rosout.h>` directly even if `qos.hpp` contains it, and the `node_base.cpp` ->  `rclcpp/node_interfaces/node_base.hpp` -> `rclcpp/callback_group.hpp` -> `rclcpp/client.hpp` -> `rclcpp/qos.hpp`.

Reply:
addressed by https://github.com/ros2/rclcpp/pull/2174/commits/b9899affc004e9233f4a7d5eeb05ef445fb2534f